engine/src/test/java/io/camunda/zeebe/engine/ArchitectureTest.java

```diff
@@ -33,5 +33,5 @@ public class ArchitectureTest {
           .resideInAPackage("io.camunda.zeebe.engine..")
           .should()
           .dependOnClassesThat()
-          .resideInAPackage("io.camunda.zeebe.scheduler..");
+          .resideInAPackage("io.camunda.zeebe.scheduler");
```

Comment:
❓ Why don't we want to restrict access to all classes? I'd prefer having no dependency on scheduler at all, if it's not necessary.